Maria/scheduler docs (#680)
telerik/maui-docs: controls/scheduler/overview.md

The Telerik UI for .NET MAUI Scheduler allows you to easily implement various scheduling scenarios in your apps built with .NET MAUI. Scheduler offers highly customizable built-in views - you can adjust current time, visible hours and dates, time ruler, special time slots and more. In addition, you have full control over the way appointments are visualized in the views.

![.NET MAUI Scheduler Overview](images/)

## Key Features

* [Built-in Views](% slug scheduler-views-overview %)&mdash;Scheduler control comes with a few customizable predefined views - month, week, multiday and day views.
* [Appointments](% slug appointments-overview %)&mdash;Through the `AppointmentsSource` property you can populate the Scheduler with appointments. In addition, you can define a custom template as well as apply various styling options to the appointments to achieve the desired look & feel.
* [Recurrent Appointments](% slug recurrence-overview %)&mdash;The control provides the functionality to configure repeating appointments. You can apply recurring scheduling patterns such as daily, weekly, monthly or set a range of recurrence from date to date to the appointments.

> * Current Date&mdash;Defines the currently visible date. Depending on the active view, `CurrentDate` specifies which day(s), week or month is visualized.

Reviewer: ```suggestion
* Current Date&mdash;Defines the currently visible date. Depending on the active view, `CurrentDate` specifies which days, week, or month are visualized.
```
